<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resetPassword</class>
 <widget class="QDialog" name="resetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="settngPasswordTitle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>397</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Resetting your Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="resetFirstNameInput">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>117</y>
     <width>176</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>91</y>
     <width>137</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter First Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="resetLastName">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>175</y>
     <width>176</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>149</y>
     <width>135</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter Last Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="resetUserName">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>233</y>
     <width>176</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>207</y>
     <width>136</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter Username </string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>561</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="resetHelpButton">
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetSubmitButton">
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>resetFirstNameInput</tabstop>
  <tabstop>resetLastName</tabstop>
  <tabstop>resetUserName</tabstop>
  <tabstop>resetHelpButton</tabstop>
  <tabstop>resetSubmitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
